doc/en/user/source/installation/docker.rst

```diff
@@ -0,0 +1,91 @@
+.. _installation_docker:
+
+Docker Container
+================
+
+Geoserver is also packaged as a Docker Container.  For more details, see the `Geoserver Docker Container Project <https://github.com/geoserver/docker>`_.
+
+Quick Start
+-----------
+
+This will run the container with the data directory shipped with the container (which is typically an empty directory).
+
+#. Make sure you have `Docker <https://www.docker.com/>`_ installed.
+#. Download the container
+
+   .. code-block:: 
+    
+      docker pull docker.osgeo.org/geoserver:2.21.1
+
+#. Run the container
+
+   .. code-block:: 
+    
+      docker run -it -p8080:8080 docker.osgeo.org/geoserver:2.21.1 
+ 
+#. Visit `http://localhost:8080/geoserver <http://localhost:8080/geoserver>`_.
+
+Using your own Data Directory
+-----------------------------
+
+This will run the container with a local data directory.  The data directory will be `mounted <https://docs.docker.com/storage/bind-mounts/>`_ into the docker container.
+
+.. Note::
+
+    Change `/MY/DATADIRECTORY` to your data directory.
```

Comment:
I find it less frustrating if we give folks a real directory to use; or use a `pwd` command.